tests/core/uc20-recovery/task.yaml

```diff
@@ -69,6 +69,25 @@ execute: |
         # restore shutdown so that spread can reboot the host
         umount /usr/sbin/shutdown
 
+        if [ "$SPREAD_BACKEND" = "external" ]; then
+            mkdir -p /home/external/.ssh
+            chown external:external /home/external/.ssh
+            touch /home/external/.ssh/rc
+            chown external:external /home/external/.ssh/rc
+            cat <<-EOF > /home/external/.ssh/rc
```

Comment:
Could you please add a reference to the `sshd(8)` manual page, specifically to the `sshrc` section.